IFC4x3/Properties/e/Emodulus_1Bctvx1L570B_eZVJbTFB8/DocProperty.xml

```diff
@@ -0,0 +1,2 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="Emodulus_1Bctvx1L570B_eZVJbTFB8" Name="Emodulus" UniqueId="4b9b7e7b-0551-4700-bfa8-8df4e574f2c8" PrimaryDataType="IfcModulusOfElasticityMeasure" />
```

Comment:
Use YoungModulus.

Reply:
> YoungModulus

As a term (property name) you mean? From my vague recollection, Young Modulus refers to tensile elasticity. @j-weil

Reply:
Fixed in [feab7eb](https://github.com/bSI-InfraRoom/IFC-Specification/pull/608/commits/feab7ebdc7ff4eb8297c47cf0d8b44f998b4018b)